<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>channelExportWindow</class>
 <widget class="QMainWindow" name="channelExportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maya to Houdini channel exporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0,0,0,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="tree_ly"/>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Add channel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="addSelectedObject_btn">
             <property name="text">
              <string>Selected objects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addFromChannelBox_btn">
             <property name="text">
              <string>From Channel Box</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addFromSet_btn">
             <property name="text">
              <string>From Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addFromFile_btn">
             <property name="text">
              <string>Import From File</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Save channels</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="saveToSet_btn">
             <property name="text">
              <string>Save To Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveToFile_btn">
             <property name="text">
              <string>Export To File</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Remove channel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="removeSelected_btn">
             <property name="text">
              <string>Remove Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeNonExists_btn">
             <property name="text">
              <string>Remove Nonexistent Objects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeAll_btn">
             <property name="text">
              <string>Clear All</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="info_lb">
          <property name="text">
           <string>Info</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Output File</string>
      </property>
      <layout class="QVBoxLayout" name="outFile_ly">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="scale_ly"/>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Frame Range</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,1,0">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="3">
          <widget class="QSpinBox" name="endRange_spb">
           <property name="minimum">
            <number>-999999999</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="startRange_spb">
           <property name="minimum">
            <number>-999999999</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Range:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QCheckBox" name="autoRange_cbx">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Request range from all animation curves in scene and set maximum and minimum to range &lt;/p&gt;&lt;p&gt;Always On in batch mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>AUTO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="currentTimeToStart_btn">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current frame to start&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>►</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="currentTimeToEnd_btn">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current frame to end&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>◄</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QPushButton" name="setTimeLineRange_btn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current animation range to export range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>▲     From Time Line    ▲ </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="batchMode_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1. Open rig scene&lt;/p&gt;&lt;p&gt;2. Load objects to list&lt;/p&gt;&lt;p&gt;3. Press Batch ...&lt;/p&gt;&lt;p&gt;4. Select fbx files&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Auto will be turn ON&lt;/p&gt;&lt;p&gt;Out clip file path will same as fbx file path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Batch...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>EXPORT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progress_pbr">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="manual_act"/>
    <addaction name="about_act"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="manual_act">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="about_act">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autoRange_cbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>startRange_spb</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoRange_cbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>endRange_spb</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
